### ISSUE TITLE
Number of training set samples

### ISSUE THREAD

**Issue by Jacobieee** [changyanchuan/TrajCL#4]
Hi, thank you for the great contribution.

I wonder why you set 200000 samples as the training set instead of the full set (above 900000 samples).

Thank you.

**Comment by Jacobieee** [comment]
Hi, I have another question about the baseline experiments.

When you were running methods like t2vec, did you also use 200k as the training set? Because I saw the reported results in your paper are different, and even better. May I know why?  And can I know how did you process the data and obtain the results for fine-tuning? I also want to test them in different datasets under your proposed evaluation pipelines.

I'll appreciate it so much for your help.

**Comment by changyanchuan** [comment]
Hi.

Yes, we use the same 200k training dataset for all self-supervised baselines (including t2vec). The results of t2vec outperforms the originally reported results in its paper may attribute to the different pre-processed datasets (I guessed)

We use another 10k dataset for fine-tuning, where the dataset is split by 7:1:2 for training, validation and test (cf. Section V.A in the paper). You may have a look of the code snippet at line 175 in utils/preprocessing_porto.py that shows how to create datasets for fine-tuning, including how to compute ground-truth labels (i.e., ground truth similarity values of heuristic measures). The Porto dataset for fine-tuning is also included. The second command in the Quick Start section in the readme give an example on how to fine-tuning the model to learn a heuristic measure.

Thanks.

**Comment by Jacobieee** [comment]
Hi,

Yes, I got your pipeline for creating labels for fine-tuning. I'm trying to run the baselines such as t2vec on other datasets and I want to test them on your setting. Would you like to provide scripts or any code snippets to run this setting on t2vec for similarity search and fine-tuning? Or some quick hints would be appreciated. You know, the deadline is approaching lol.

Thank you so much!

**Comment by changyanchuan** [comment]
Hi,

The overall idea is as follows.
I used t2vec's original implementation to pre-train it. To align the pre-training datasets, I converted the pre-training datasets for TrajCL to t2vec based on its required data format. Once t2vec is pre-trained (by using t2vec's code), I used TrajCL code to fine-tune t2vec. Thus, t2vec and TrajCL classes have same signatures for forward, interpret and checkpoint functions. 

I just quickly viewed the changes I made on t2vec. It is roughly ~500 line changes. It is hard to attach all files here. Besides, I added some code to adapt t2vec in TrajCL, including t2vec implementation for fine-tuning, pre-training dataset preparation. 

A lot of code for the whole project, for simplicity, so I won't provide it here. Thank you for understanding.
